### PR TITLE
Retrieve data about Mastodon servers

### PR DESCRIPTION
**Pull request by annando**
This is a replacement for the old API that isn't available any more.

### CODE REVIEW

**Comment by MrPetovan** [comment]
This PR needs a link to https://instances.social/api/token in the doc.

**Comment by annando** [comment]
Okay?

File: doc/htconfig.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Example: To set the directory value please add this line to your .htconfig.php:
 * **frontend_worker_timeout** - Value in minutes after we think that a frontend task was killed by the webserver. Default value is 10.
 * **hsts** (Boolean) - Enables the sending of HTTP Strict Transport Security headers
 * **ignore_cache** (Boolean) - For development only. Disables the item cache.
-* **instances_social_key** - Key to the API of https://instances.social which retrieves data about mastodon servers.
+* **instances_social_key** - Key to the API of https://instances.social which retrieves data about mastodon servers. See https://instances.social/api/token for getting an API key.
```
Comment (MrPetovan) [inline doc/htconfig.md]: `to get` instead of `for getting`.

Comment (annando) [reply doc/htconfig.md]: Oui, Monsieur le tatillon. 😀

**Pull request merged**